pkg/kubelet/kubelet.go

```diff
@@ -2075,15 +2075,15 @@ func (kl *Kubelet) HandlePodCleanups() error {
 	err = kl.cleanupOrphanedPodDirs(allPods, runningPods)
 	if err != nil {
 		glog.Errorf("Failed cleaning up orphaned pod directories: %v", err)
-		return err
```

Comment:
nit: Add a comment stating that we want all cleanup tasks to be run even if one of them failed.